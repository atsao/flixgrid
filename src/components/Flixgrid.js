import React from 'react';

import Table from './Table';

const Flixgrid = props => <Table {...props} />;

export default Flixgrid;
