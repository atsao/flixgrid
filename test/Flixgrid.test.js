import React from 'react';
import renderer from 'react-test-renderer';

import Flixgrid from '../src/components/Flixgrid';

it('renders a Flixgrid component', () => {
  const tree = renderer.create(<Flixgrid />);
  expect(tree).toMatchSnapshot();
});
