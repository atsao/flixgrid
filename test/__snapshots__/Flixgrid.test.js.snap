// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Flixgrid component 1`] = `
<table>
  <tbody>
    <tr>
      <td>
        nothing
      </td>
    </tr>
  </tbody>
</table>
`;
